src/Microsoft.DotNet.ProjectJsonMigration/ConstantPackageNames.cs

```diff
@@ -1,10 +1,29 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections.Generic;
+
 namespace Microsoft.DotNet.ProjectJsonMigration
 {
     internal class ConstantPackageNames
     {
         public const string CSdkPackageName = "Microsoft.NET.Sdk";
+
+        public static readonly IDictionary<string, string> AspProjectDependencyToolsPackages = new Dictionary<string, string> {
```

Comment:
I don't think so. We need to move these packages to their "final" migration versions.